projects/hmcts-annotation-ui-lib/src/lib/components/comments/comment-item/comment-item.component.ts

```diff
@@ -132,4 +161,17 @@ export class CommentItemComponent implements OnInit, OnDestroy {
             this.hideButton = true;
         }, 100);
     }
+
+    getRelativePosition(annotationId: string): number {
+      const svgSelector = this.document.querySelector(`g[data-pdf-annotate-id="${annotationId}"]`);
+      const highlightRect = <DOMRect>svgSelector.getBoundingClientRect();
+
+      const wrapper = this.document.querySelector('#annotation-wrapper');
+      const wrapperRect = <DOMRect>wrapper.getBoundingClientRect();
+
+      const top = ((highlightRect.y - wrapperRect.top)
+        - 59) - 5; // Minus height of toolbar + 5px
```

Comment:
Try removing the -59 and -5. The top of the highlight should be aligned with the top of the comment box